lib/bandit/http1/adapter.ex

```diff
@@ -159,6 +159,7 @@ defmodule Bandit.HTTP1.Adapter do
   # `close` & `keep-alive` always means what they say, otherwise keepalive if we're on HTTP/1.1
```

Comment:
```suggestion
  # `close` & `keep-alive` always means what they say, otherwise keepalive if we're on HTTP/1.1
  # Case insensitivity per RFC9110§7.6.1
```